liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/InstallFeatureSupport.java

```diff
@@ -152,6 +169,20 @@ protected List<String> getAdditionalJsonList() {
         }
         return result;
     }
+    
+    protected Collection<Map<String, String>> getKeyMap(){ 	
+    	Collection<Map<String,String>> keyMapList = new ArrayList<>(); 
+    	if(keys != null) {
+    		for(Key k: keys.values()) {
+    			Map<String, String> keyMap = new HashMap<>();
+        		getLog().info("Key Id: " + k.getKeyid() +" Key URL: " + k.getKeyurl());
```

Comment:
Is it ok to print these to the log? If so, should they be debug level instead of info?